runsc/boot/restore.go

```diff
@@ -169,8 +169,9 @@ func (r *restorer) restore(l *Loader) error {
 		return fmt.Errorf("creating platform: %v", err)
 	}
 
-	// Start the old watchdog before replacing it with a new one below.
+	// Start and Stop the old watchdog before replacing it with a new one below.
 	l.watchdog.Start()
+	l.watchdog.Stop()
```

Comment:
I think it should be stopped on line 246, right before replacing `l.watchdog` with the new watchdog.

Reply:
Thanks for pointing out, already changed.

Reply:
LGTM